crates/nu-command/src/viewers/explore.rs

```diff
@@ -368,3 +371,39 @@ fn convert_json_value_into_value(value: nu_json::Value) -> Value {
         }
     }
 }
+
+fn include_nu_config(config: &mut HashMap<String, Value>, style_computer: &StyleComputer) {
+    let line_color = lookup_color(style_computer, "separator");
+    if line_color != nu_ansi_term::Style::default() {
+        {
+            let mut map = config
+                .get("table")
+                .and_then(parse_hash_map)
+                .unwrap_or_default();
+            insert_style(&mut map, "split_line", line_color);
+            config.insert(String::from("table"), map_into_value(map));
+        }
+
+        {
+            let mut map = config
+                .get("try")
+                .and_then(parse_hash_map)
+                .unwrap_or_default();
+            insert_style(&mut map, "border_color", line_color);
+            config.insert(String::from("try"), map_into_value(map));
+        }
+
+        {
+            let mut map = config
+                .get("config")
+                .and_then(parse_hash_map)
+                .unwrap_or_default();
+            insert_style(&mut map, "border_color", line_color);
+            config.insert(String::from("config"), map_into_value(map));
+        }
+    }
+}
+
+fn lookup_color(style_computer: &StyleComputer, key: &str) -> nu_ansi_term::Style {
+    style_computer.compute(key, &Value::nothing(Span::unknown()))
```

Comment:
Since this is a one-liner only used in 1 function, I'd personally do it as just a macro inside include_nu_config()… but whatever.